src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -56,6 +59,7 @@ class TypeRelaxedExtension : public ov::OpExtension<ov::op::TypeRelaxed<Op>> {
     }
 
 private:
+    std::string m_version;
```

Comment:
In fact, we don't need to store the `m_version` per object.
We can do the following:

```cpp
template <typename Op>
class TypeRelaxedExtension : public ov::OpExtension<ov::op::TypeRelaxed<Op>> {
private:
static const char* version() {
    static const std::string version = std::string(ov::op::TypeRelaxedBase::version_prefix) + Op::get_type_info_static().version_id;
    return version.data();
}

public:
TypeRelaxedExtension() : m_ext_type(Op::get_type_info_static().name, version()) {}
```
In that way we initialize a static object and don't spawn a lot of equal `std::strings`

Reply:
Applied.